src/index.js

```diff
@@ -6,8 +6,17 @@ import shouldUpdate from './shouldUpdate'
 
 const noop = () => {}
 
+const defaultClasses = {
+  main: 'headroom',
+  unfixed: 'headroom--unfixed',
+  scrolled: 'headroom--scrolled',
```

Comment:
`classes.scrolled` is never applied, it should be applied https://github.com/KyleAMathews/react-headroom/blob/3deb42b3f3ba8ebf7054e188937dc047c6080e67/src/index.js#L298